doc/source/merging.rst

```diff
@@ -817,6 +817,10 @@ equivalent:
     left.join(right, on=key_or_keys)
     pd.merge(left, right, left_on=key_or_keys, right_index=True,
           how='left', sort=False)
+          
+.. warning::
+
+ * Joining on index with duplicate keys when joining large dataframes would cause severe memory overflow, sometimes freezes the            computer and user has to hard reboot, which can be dangerous for unsaved work. Please make sure no duplicate keys in index              before joining.
```

Comment:
It's not needed to start with `*` (as this will give a one-bullet list). You only need an indentation so the text belongs to the warning. 
Further, you have some wonky spaces in the text.

On the content: I don't think this is specific to joining on the index? But just on joining on a key with duplicate values?